packages/maskbook/src/extension/background-script/HelperService.ts

```diff
@@ -64,3 +64,7 @@ export async function requestBrowserPermission(permission: browser.permissions.P
         })
     })
 }
+
+export function queryPermission(permission: browser.permissions.Permissions) {
+    return browser.permissions.contains(permission)
```

Comment:
This can be called in the content script safely. There is no need to wrap it in a service

Reply:
Oh then I might be doing something wrong because when I call it from the PostDialog.tsx component I get `TypeError: Cannot read property 'contains' of undefined` and `browser.permissions` is actually `undefined`

Reply:
Maybe I recall that wrong 🤔